core/txpool/legacypool/fastcache.go

```diff
@@ -0,0 +1,187 @@
+package legacypool
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/VictoriaMetrics/fastcache"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type LRUBufferFastCache struct {
+	cache    *fastcache.Cache
```

Comment:
https://pkg.go.dev/github.com/VictoriaMetrics/fastcache#section-readme
```
Big entries with sizes exceeding 64KB must be stored via [distinct API](http://godoc.org/github.com/VictoriaMetrics/fastcache#Cache.SetBig).
```

so it's not suitable for blob tx?